constants/users.js

```diff
@@ -0,0 +1 @@
+module.exports = ['ooo', 'idle', 'active']
```

Comment:
Can we call this list as something like `userStatusTypes` and then export that list? Because if a third person visits this file, they might not be able to make sense of what this list in doing in users constants, I hope that makes sense 😄

Reply:
I have updated the constants/users.js file as you suggested!

https://github.com/Real-Dev-Squad/website-backend/pull/348/commits/76d5c2900c4f50882b2db2e6fb879b76231885e9